content/contributing/documentation/rst_cheat_sheet.rst

```diff
@@ -734,6 +750,62 @@ Render
              public static void main(String[] args) {}
          }
 
+.. _contributing/tabs/code:
+
+Code tabs
+---------
+
+Code tabs are essentially :ref:`group tabs <contributing/tabs/group>` that treat the content as a
+code block. The `code-tab` directive is used to define a code tab. Just as for the `code-block`
```

Comment:
`code-block` isn't documented on this page, yet you reference it.

Reply:
Now it is!